examples/apt.py

```diff
@@ -3,10 +3,49 @@
 
 SUDO = True
 
-if host.fact.linux_name in ['Debian', 'Ubuntu']:
+# Note: Using linux_distribution fact so running from docker
+# will show valid name, otherwise could just use host.fact.linux_name
+linux_distribution = host.fact.linux_distribution
```

Comment:
This could all be written as:

```py
code_name = host.fact.linux_distribution['release_meta'].get('DISTRIB_CODENAME')

if host.fact.linux_name...
```

The `linux_name` fact actually uses `linux_distribution` under the hood - so this won't cause any double load.

Reply:
Thanks